com.unity.render-pipelines.core/Editor/Common/Settings/SerializedScalableSettingValue.cs

```diff
@@ -1,14 +1,14 @@
 using System;
 using UnityEngine;
 using UnityEngine.Assertions;
-using UnityEngine.Rendering.HighDefinition;
+using UnityEngine.Rendering;
 
-namespace UnityEditor.Rendering.HighDefinition
+namespace UnityEditor.Rendering
 {
     /// <summary>
     /// Serialized version of <see cref="ScalableSettingValue{T}"/>.
     /// </summary>
-    internal class SerializedScalableSettingValue
+    public class SerializedScalableSettingValue
     {
         public SerializedProperty level;
         public SerializedProperty useOverride;
```

Comment:
This public field need to be private with public getter only. And documentation is needed